bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLoggerAllocator.java

```diff
@@ -90,16 +92,19 @@ synchronized long getPreallocatedLogId() {
         return preallocatedLogId;
     }
 
+    public boolean isSealed(long logId) {
+        return logId != writingLogId && logId != writingCompactingLogId;
+    }
```

Comment:
Isn't this misleading if it is called "isSealed"?

Reply:
`isSealed` means that the entry log file will no longer be written to, we needn't care that the file sile change.